internal/wire/parse.go

```diff
@@ -214,6 +218,21 @@ type InjectorArgs struct {
 	Pos token.Pos
 }
 
+// Field describes a list of fields from a struct.
+type Field struct {
+	// Parent is the struct the field belongs to.
```

Comment:
Worth noting that the underlying type is always either a struct or a pointer to a struct.